commands/service_dev.go

```
@@ -63,14 +63,17 @@ func (c *serviceDevCmd) runE(cmd *cobra.Command, args []string) error {
 			errors.New("To get more information, run: mesg-core service validate"),
 		}
 	}
-	fmt.Printf("%s Service deployed with ID: %v\n", pretty.SuccessSign, pretty.Success(id))
-	defer pretty.Progress("Removing service...", func() { c.e.ServiceDelete(id) })
+	fmt.Printf("%s Service deployed with ID: %v\n\t", pretty.SuccessSign, pretty.Success(id))
+	defer func() {
+		pretty.Progress("Removing the service...", func() { c.e.ServiceDelete(id) })
+		fmt.Printf("%s Service removed.\n", pretty.SuccessSign)
```

ServiceDelete err is not checked here